api/iscsi.proto

```diff
@@ -0,0 +1,364 @@
+syntax = "proto3";
+
+package api;
+
+import "errors.proto";
+
+service IscsiCSIProxyService {
```

Comment:
Can we also have [AddIScsiConnection](https://docs.microsoft.com/en-us/windows/win32/api/iscsidsc/nf-iscsidsc-addiscsiconnectionw) here?

Reply:
Adding